### PR TITLE
set the skeletondirectory by using the testing app

### LINKED ISSUE #34483: skeleton dir is wrongly guessed when running tests on remote systems
`run.sh` does guess the place of the skeleton folder by searching for it on the system that runs the tests, that information might not be correct on the SUT if that is a truly remote system

### PR DESCRIPTION
**Pull request by individual-it**
## Description
don't try to guess and set the skeleton directory from outside of the SUT use the new endpoint of the testingapp for that 
needs https://github.com/owncloud/testing/pull/88

## Related Issue
- Fixes #34483

## Motivation and Context
guessing the correct skeleton directory by the test runner is not reliably, we need to let the SUT do it itself

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/run.sh [hunk 1]
```diff
@@ -740,6 +713,29 @@ do
 	done
 done
 
+#set the skeleton folder
+if [ -z "${SKELETON_DIR}" ]
+then
+	# calculate the correct skeleton folder
+	for URL in ${TESTING_APP_URL} ${TESTING_APP_FED_URL}
+	do
+		if [ "${RUNNING_API_TESTS}" = true ] || [ "${RUNNING_CLI_TESTS}" = true ]
```
Comment (phil-davis) [inline tests/acceptance/run.sh]: This if-then-else block could go outside the `for` loop.

Comment (individual-it) [reply tests/acceptance/run.sh]: moved the block

**Review by phil-davis — DISMISSED** [review]
Looks OK if CI passes

**Comment by individual-it** [comment]
some methods used to get the local file from the skeleton folder and compare a downloaded file with it. Now we need to download the file from the remote skeleton before comparing

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
backport in  #34573

**Pull request merged**